addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -228,6 +248,22 @@
                         t-on-click="(ev) => this.onCellClicked(record, column, ev)" tabindex="-1">
                         <t t-if="!evalModifier(column.modifiers.invisible, record)">
                             <t t-if="canUseFormatter(column, record)" t-out="getFormattedValue(column, record)"/>
+                            <t t-elif="column.propertyType">
+                                <PropertyValue t-if="record.data[column.name] and record.data[column.name].value"
+                                    canChangeDefinition="false"
+                                    context="record.context"
+                                    comodel="record.data[column.name].comodel"
+                                    domain="this.fields[column.name].domain"
```

Comment:
Shouldn't it be `record.data[column.name].domain` ? (the domain here come from the definition)